client/checkout/woopay/direct-checkout/woopay-direct-checkout.js

```diff
@@ -14,6 +14,11 @@ import SessionConnect from 'wcpay/checkout/woopay/connect/session-connect';
 class WoopayDirectCheckout {
 	static userConnect;
 	static sessionConnect;
+	static redirectElements = {
+		CLASSIC_CART_PROCEED_BUTTON: '.wc-proceed-to-checkout .checkout-button',
+		BLOCKS_CART_PROCEED_BUTTON:
+			'.wp-block-woocommerce-proceed-to-checkout-block',
+	};
```

Comment:
This is for having a single source for all elements that will trigger the direct checkout feature.